perfmetrics/scripts/run_e2e_tests.sh

```diff
@@ -53,28 +53,120 @@ sudo apt-get install -y gcc python3-dev python3-setuptools
 sudo apt install -y python3-crcmod
 
 # Create bucket for integration tests.
-# The prefix for the random string
-bucketPrefix="gcsfuse-integration-test-"
-# The length of the random string
-length=5
-# Generate the random string
-random_string=$(tr -dc 'a-z0-9' < /dev/urandom | head -c $length)
-BUCKET_NAME=$bucketPrefix$random_string
-echo 'bucket name = '$BUCKET_NAME
-# We are using gcloud alpha because gcloud storage is giving issues running on Kokoro
-gcloud alpha storage buckets create gs://$BUCKET_NAME --project=$PROJECT_ID --location=$BUCKET_LOCATION --uniform-bucket-level-access
+function create_bucket() {
```

Comment:
Instead pass take bucket_prefix as an argument or $1 and create bucket based on that.